src/impl/realm_coordinator.hpp

```diff
@@ -21,6 +21,8 @@
 
 #include "shared_realm.hpp"
 
+#include <realm/version_id.hpp>
```

Comment:
Needed because it's no longer pulled in by shared_realm.hpp. `VersionID` is used in a few places in this file.